site/en/blog/chromium-chronicle-25/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "The Chromium Chronicle #25: Thread Safety Annotations"
+description: >
+  Learn how Clang's static analysis framework takes over the toil of thread-safety proofs.
+layout: 'layouts/blog-post.njk'
+date: 2021-09-27
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 25:** by Victor Constan in SFO (September, 2021)<br>
```

Comment:
Constan -> Costan

(sorry for the late comment, I just noticed this)

Reply:
```suggestion
**Episode 25:** by Victor Costan in SFO (September, 2021)<br>
```